config/samples/backends/cinder/glance-common/glance.yaml

```diff
@@ -25,10 +25,7 @@ spec:
       databaseInstance: openstack
       glanceAPIs:
         default:
-          debug:
-            service: false
-          preserveJobs: false
           replicas: 1
-          type: split
+          type: single
```

Comment:
I think `split` is required for `cinder` in general, right? 
I'm using `single` only for `File` || `NFS` backend, so this field should be reverted I guess

Reply:
++ split is required

Reply:
This is covered by: https://github.com/openstack-k8s-operators/glance-operator/pull/459 where we updated and fixed (for the very same reason) most of the samples.

Reply:
What does it mean `split` is required?
As far as I know it is not required, it's an option, right?
It is an option to solve the internal vs public API issue.